### ISSUE TITLE
Colors are way too bright on MacBook Pro

### ISSUE THREAD

**Issue by asilvadesigns** [Microsoft/vscode#50349]
At work and home I often use non retina monitors, and this issue makes it very difficult, and at the least painful to work from my laptop. Atom doesn't handle colors like this, nor any other app I've noticed. Please see screenshot below:

Atom is on the left, VS Code is on the right... (notice the hex value in the dev tools)
_Screenshot taken on a 2018 MacBook Pro._

The hex values are identical, but VS Code renders then wayyy brighter. I love working in VS Code, so hopefully this is a simple change on my end.

<img width="1434" alt="screen shot 2018-05-23 at 8 10 49 am" src="https://user-images.githubusercontent.com/7071307/40423584-09f36c38-5e61-11e8-810d-4eefe4da4ef2.png">

- VSCode Version: 1.24.0-insider (occurs with non-insider build as well)
- OS Version: Mac OS 10.12.6

Steps to Reproduce:

1. Open any theme on a new MacBook Pro.
2. Notice the over saturated, painfully bright colors.

Does this issue occur when all extensions are disabled?: Yes

**Comment by CodigoMatriz** [comment]
It's happening to me as well. 
The colors are way too bright on my MacBook Pro 2015 Retina.

**Comment by asilvadesigns** [comment]
I've noticed since electron 2.0, which I think is as of the last day or two, this has been resolved for me.
@CodigoMatriz @bpasero can you confirm?

**Comment by bpasero** [comment]
@asilvadesigns can you give [1] a try which is using Electron 2.0.x to see if that makes it any better?

[1] https://az764295.vo.msecnd.net/insider/c13b8641174a66e6316d809c785fdc884998bc89/VSCode-darwin-insider-unsigned.zip

**Comment by sgon00** [comment]
+1

VSCode: 1.25.1
OS: 10.13.6

I tried many themes. The colors in all different themes are much much more bright than the colors in my Linux.

**Comment by sgon00** [comment]
sorry I need to add this: not only the **colors** are different, but also the fonts are **more bold**. I am unable to read texts in such extremely **bright** and extremely **bold** situation.  I just switched from Linux vscode to MacOS vscode. This is just crazy.

**Comment by MicrosoftEurope** [comment]
Hello sgon00,

    Could you please attach a screenshot ?

вс, 5 авг. 2018 г., 12:49 sgon00 <notifications@github.com>:

**Comment by sgon00** [comment]
@MicrosoftEurope sorry about the reply a bit late. I am still trying to get used to the new system. Never use MacOS before, even don't know how to make a screenshot. I did adjust contrast and brightness after the last post, it looks much better now even if it's still more bright and more bold. Please check the screenshots:

![vscode-linux](https://user-images.githubusercontent.com/1555508/43685060-91d23276-98de-11e8-824f-51340b360e87.png)
![vscode-mac](https://user-images.githubusercontent.com/1555508/43685062-95ee1a32-98de-11e8-9915-5cfe0e911a9d.png)

Btw, the font is the same, but the lineHeight is different. I forgot to make them the same.

**Comment by sgon00** [comment]
hi, all, FYI, I found the solution (at least for me). Simply add the following line to `User Settings`:

```
  "workbench.fontAliasing": "antialiased",
```

Both built-in monitor and external monitor look very well with this setting. The font no longer looks bright and bold anymore.

**Comment by leandrocunha** [comment]
@sgon00   "workbench.fontAliasing": "antialiased" is a unknown configuration setting.

**Comment by sgon00** [comment]
@leandrocunha I have no idea what you are talking about. This setting just works very well. Please check the following screenshot and check `DEFAULT USER SETTINGS` and my own `USER SETTINGS`:

![screen shot 2018-09-08 at 11 36 29 am](https://user-images.githubusercontent.com/1555508/45249940-a5365f80-b35b-11e8-9c38-c4862bc1bbea.png)

**Comment by leandrocunha** [comment]
@sgon00 I think you are using a old version of VSCode, currently this setting is not available. What is the version your are using?

**Comment by sgon00** [comment]
@leandrocunha Your thoughts are wrong. I am using the latest build. 

```shell
$ code --version
1.27.1
5944e81f3c46a3938a82c701f96d7a59b074cfdc
x64
```

![screen shot 2018-09-08 at 11 44 57 pm](https://user-images.githubusercontent.com/1555508/45255935-84a0f080-b3c1-11e8-8524-53a97c4c7339.png)

![screen shot 2018-09-08 at 11 49 38 pm](https://user-images.githubusercontent.com/1555508/45255956-ea8d7800-b3c1-11e8-928d-252b3eafb505.png)

**Comment by vedatmahir** [comment]
https://github.com/Microsoft/vscode/issues/50349#issuecomment-402746499 @bpasero 
Looks like there is no brightness problem in the Insider version. Thanks!
You can see the difference from below.

<img width="1920" alt="screen shot 2018-09-08 at 19 59 03" src="https://user-images.githubusercontent.com/5601908/45256616-aee2b600-b3a1-11e8-80e6-dd89590e3653.png">

**Comment by bpasero** [comment]
@vedatmahir yeah we pushed a change for 1.27, can you update Code stable to 1.27.1 and verify the fix is there as well?

**Comment by vedatmahir** [comment]
@bpasero I'm already using that (1.27.1) version. Please look at the screenshot I sent in my previous comment.

**Comment by leandrocunha** [comment]
@sgon00 Crazy man. Take a look at my printscreen
![screenshot from 2018-09-08 22-04-39](https://user-images.githubusercontent.com/1820920/45260073-6a601600-b3b3-11e8-9c4b-c5d40e603d1a.png)

**Comment by bpasero** [comment]
@asilvadesigns can you please comment if this issue is resolved for you?

**Comment by sgon00** [comment]
@leandrocunha ok, I am **very sorry** for the bad words that I posted too. Sorry that I was too emotional because I never saw a thumbs down emoji on my comments in github before.

**Comment by leandrocunha** [comment]
@sgon00 Don't worries, I should pay attention in the issue's title. I read the solution with the little bit emotional dose, excited to solve the problem, because I hate work in Linux :smile:

**Comment by sgon00** [comment]
@leandrocunha 👍👍😃😃🤝🤝

**Comment by bpasero** [comment]
Timeout.

**Comment by lfaoro** [comment]
@bpasero why did you close this if it is not fixed? I'm on `1.27.2` and experiencing the same issue.

**Comment by bpasero** [comment]
@lfaoro we close issues without reply from the author after some days. 

Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/
